.elpaignore

```diff
@@ -1 +1 @@
-embark-org.el
\ No newline at end of file
+LICENSE
```

Comment:
Wait, is ignoring the LICENSE correct?

Reply:
Or am I going to get a distressed email from Stefan about removing the LICENSE file? :)

Reply:
Yes, it is fine. Stefan added many LICENSE ignores in elpa-packages himself.

Reply:
OK, good. I thought you probably knew for a fact this was OK, but wanted to confirm just in case.